### ISSUE TITLE
Tablet setup

### ISSUE THREAD

**Issue by Aqua1ung** [linuxwacom/libwacom#616]
First of all, thank you all for maintaining this project! One very grateful user here.

Got me a Wacom tablet (CTL-472), and so far things work as planned in Linux. Here are a few questions for you guys:

1. How can I set up the tablet to be used upside down? USB port is on the left side, which is horribly inconvenient, hence I want it on the right side;
2. Is there a tool that can map the tablet to just a portion of the screen, preferably GUI-based?

**Comment by whot** [comment]
1) easy enough, enable left-handed mode (e.g. gnome control center, the wacom panel), not sure what it's called in other DEs. `Option "Rotate"` if you're using X11 and want to set it via the xorg.conf (but that will be overridden by the DE anyway)
2) only under X11 and even there I'm not aware of a GUI. There's some info on the [xf86-input-wacom wiki](https://github.com/linuxwacom/xf86-input-wacom/wiki/Dual-and-Multi-Monitor-Set-Up). I think KWin *may* support area mapping in Wayland but I'm not 100% sure.

**Comment by Aqua1ung** [comment]
> 1. easy enough, enable left-handed mode (e.g. gnome control center, the wacom panel), not sure what it's called in other DEs. `Option "Rotate"` if you're using X11 and want to set it via the xorg.conf (but that will be overridden by the DE anyway)

Yeah, I tried that. "Left Hand Orientation" has absolutely no effect on my setup unfortunately.
![Screenshot from 2023-12-14 13-20-34](https://github.com/linuxwacom/libwacom/assets/12104779/bcbb413d-7953-4455-8859-245539a85105)

**Comment by whot** [comment]
can you try `sudo libinput debug-events --enable-left-handed` please? Then check the coordinates as they come out of libinput for the various tablet tool events, they should be upside-down, i.e. bottom right of the tablet should be 0/0. Is that the case?

If not, it's a libinput bug. If it is then the bug is in GNOME somewhere that it doesn't trigger the setting. If you're on X11 make sure the xf86-input-wacom driver is installed (xorg-x11-drv-wacom on Fedora).

**Comment by Aqua1ung** [comment]
@whot: here's what I get with ["Left Hand Orientation" active](https://i.imgur.com/jdma0Bd.png):
```
bottom left: event21  TABLET_TOOL_AXIS        +3.570s		0.00/94.97	pressure: 0.44*
bottom right: event21  TABLET_TOOL_AXIS      +27.638s	151.97/94.97	pressure: 0.26*
```
OI'm guessing it's a libinput bug, is it?

**Comment by whot** [comment]
> By your description, this looks like a libinput bug. 

yep, libinput bug. Fixed in https://gitlab.freedesktop.org/libinput/libinput/-/merge_requests/962

**Comment by Aqua1ung** [comment]
> yep, libinput bug. Fixed in https://gitlab.freedesktop.org/libinput/libinput/-/merge_requests/962

Did this fix find its way in 1.25.0?

**Comment by whot** [comment]
yes, it was the last patch to go in
